src/src/ex7/client/src/components/SearchBox/__tests__/SearchBox.test.jsx

```diff
@@ -0,0 +1,9 @@
+import renderer from "react-test-renderer";
+import SearchBox from "../SearchBox";
+
+test("Search box response properly", ()=>{
```

Comment:
worth adding some functionality testing regarding the component, such as actions and styling